client/src/contexts/boardContext.js

```diff
@@ -1,109 +1,15 @@
 import React, {createContext, useEffect, useReducer, useState} from 'react';
 import Board from 'api/Board';
-import {updateColumn} from 'api/Column';
+import reducers from './boardReducers';
```

Comment:
This is great. It was a lot to look at beforehand in a single file. Now it's clear where the reducers live.